scripts/update-ts-configs.js

```diff
@@ -99,23 +100,23 @@ function writeRootTsConfigJson(pkgRoot, projectRoot, lernaProjects) {
   const tsconfig = readJSON(tsconfigPath);
   const references = Array.from(lernaProjects.values())
     .filter(it => it.isTsProject)
-    .map(it => path.relative(pkgRoot, path.join(projectRoot, it.dir))).sort();
+    .map(it => toPosix(path.relative(pkgRoot, path.join(projectRoot, it.dir)))).sort();
   tsconfig.references = references.map(path => {
-    return { path }
+    return { path: toPosix(path) }
   });
   tsconfig.typedocOptions.entryPoints = Array.from(lernaProjects.values())
     .filter(it => !it.private && it.isTsProject)
-    .map(it => path.relative(pkgRoot, path.join(projectRoot, it.dir))).sort();
+    .map(it => toPosix(path.relative(pkgRoot, path.join(projectRoot, it.dir)))).sort();
   writeJSON(tsconfigPath, tsconfig, dryRun);
 
   for (const tsconfigName of ['tsconfig.esm.json', 'tsconfig.esnext.json']) {
     const tsconfigPath = path.join(pkgRoot, tsconfigName);
     const tsconfig = readJSON(tsconfigPath);
     const references = Array.from(lernaProjects.values())
       .filter(it => it.isTsProject && it.hasMultiTarget)
-      .map(it => path.relative(pkgRoot, path.join(projectRoot, it.dir))).sort();
+      .map(it => toPosix(path.relative(pkgRoot, path.join(projectRoot, it.dir)))).sort();
```

Comment:
Duplicated with line 119.

```suggestion
      .map(it => path.relative(pkgRoot, path.join(projectRoot, it.dir))).sort();
```

Reply:
to late - @dyladan merged already...